Client.Linq/Internal/Expressions/String/StartsWith.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions.String
+{
+    internal class StartsWith : IExpressionPart
+    {
+        private readonly IEnumerable<IExpressionPart> expressionParts;
+        private readonly IEnumerable<IExpressionPart> expressionParts2;
+
+        internal StartsWith(IEnumerable<IExpressionPart> expressionParts, IEnumerable<IExpressionPart> expressionParts2)
+        {
+            this.expressionParts = expressionParts;
+            this.expressionParts2 = expressionParts2;
+        }
+
+        public void AppendFlux(StringBuilder builder)
+        {
+            builder.Append("strings.hasPrefix(v: ");
+            foreach (var expressionPart in expressionParts) expressionPart.AppendFlux(builder);
+            builder.Append(", prefix: ");
+            foreach (var expressionPart in expressionParts2) expressionPart.AppendFlux(builder);
+            builder.Append(")");
+        }
```

Comment:
Please use same naming as is in Flux documentation: https://docs.influxdata.com/flux/v0.x/stdlib/strings/hasprefix/.